Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -194,6 +194,41 @@ public extension Notification.Name {
      */
     static let routeControllerDidFailToReroute: Notification.Name = .init(rawValue: "RouteControllerDidFailToReroute")
     
+    /**
+     Posted when `RouteController` has detected a change in alternative routes list.
+     
+     The user info dictionary contains `RouteController.NotificationUserInfoKey.locationKey` and `RouteController.NotificationUserInfoKey.removedAlternativesKey` keys.
+     */
+    static let routeControllerDidUpdateAlternatives: Notification.Name = .init(rawValue: "routeControllerDidUpdateAlternatives")
```

Comment:
Should this and other notification names end in `AlternativeRoutes` for consistency with `AlternativeRoute`? (Not a huge deal either way.)